wp-content/plugins/jetpack/_inc/build/contact-form/js/grunion-admin.min.asset.php

```diff
@@ -0,0 +1 @@
+<?php return array('dependencies' => array(), 'version' => 'aeac00bf3d7f0899a2588db74a028130');
```

Comment:
:no_entry_sign: **Error( severity 5 )**: When a multi-item array uses associative keys, each value should start on a new line (*WordPress.Arrays.ArrayDeclarationSpacing.AssociativeArrayFound*).